.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+dist: bionic
+before_install:
+  - sudo apt-get update
+  - sudo apt-get -y install gnupg2 libgpgme-dev swig
+install:
+  - pip install --upgrade poetry
+  - poetry install
+matrix:
+  include:
+  - name: "Lint and static analysis"
+    python: 3.6
```

Comment:
Any particular reason not to run lint / static analysis in a newer Python version?

Reply:
nope!